src/ui/categories/common/CategoryTreeSelectorRow.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { Button, Label } from 'patternfly-react';
+
+import TreeNodeFolderIcon from 'ui/common/tree-node/TreeNodeFolderIcon';
+import TreeNodeExpandedIcon from 'ui/common/tree-node/TreeNodeExpandedIcon';
+import RowSpinner from 'ui/common/RowSpinner';
+
+class CategoryTreeSelectorRow extends Component {
+  renderTags() {
```

Comment:
there is really no need for this to be an actual method of the class.
you can just have the `.map` inside the render method and assign that to a constant 

as a rule of thumb the only method in a class returning JSX should be render and that's it.
if you feel something like this is necessary it probably means you should be creating a new component, which is probably the better option here